Automattic/wp-calypso
# Checkout: remove annual features from the "what's included" list when picking a monthly plan on tailored flows

Related to https://github.com/Automattic/wp-calypso/pull/75679.

## Proposed Changes

The tailored flows' checkout sidebar currently includes annual plan features even though the picked plan is monthly-termed. This PR fixes that.

## Testing Instructions

Test the Link-in-bio flow (`/setup/link-in-bio`), Hosting onboarding flow (`/start/hosting`), and regular onboarding flow (`/start`):

1. Start the flow and verify that switching to the monthly plans view strikes the annual features (ss 1);
2. Proceed to checkout and check that the annual features (e.g. free domain included) are not on the sidebar list (ss 2);
3. Start the flow again and check that the annual features appear on the list if you select an annual plan.

## Screenshots

| 1 | 2 |
| - | - |
| <img width="519" alt="image" src="https://user-images.githubusercontent.com/26530524/232074311-c61e8ed4-81e0-4236-9296-ddb8c75525b6.png"> | <img width="929" alt="image" src="https://user-images.githubusercontent.com/26530524/232074512-3270d35b-d7c2-43b3-8164-b8bc97817d30.png"> |
